src/backends/pyplot.jl

```diff
@@ -229,11 +229,6 @@ function fix_xy_lengths!(plt::Plot{PyPlotBackend}, series::Series)
     end
 end
 
-py_linecolor(series::Series)          = py_color(series[:linecolor])
-py_markercolor(series::Series)        = py_color(series[:markercolor])
-py_markerstrokecolor(series::Series)  = py_color(series[:markerstrokecolor])
-py_fillcolor(series::Series)          = py_color(series[:fillcolor])
-
 py_linecolormap(series::Series)       = py_colormap(series[:linecolor])
```

Comment:
I could not understand how `py_linecolormap` should be replaced

Reply:
Yeah, leave that as it is.